src/main/kotlin/calculadoraJUnit/Calculadora.kt

```diff
@@ -0,0 +1,20 @@
+package calculadoraJUnit
+
+class Calculadora {
+
+    fun soma(x: Int, y: Int): Int {
+        return x + y
+    }
+
+    fun subtracao(x: Int, y: Int): Int {
+        return x - y
+    }
+
+    fun multiplicacao(x: Int, y: Int): Int {
+        return x * y
+    }
+
+    fun divisao(x: Int, y: Int): Int {
+        return x / y
+    }
+}
```

Comment:
```suggestion
}

```
![image](https://user-images.githubusercontent.com/53612103/185622558-2e62c898-6a75-4410-bb0c-d4078dd018f4.png)